benchmark/unified_benchmark.py

```
@@ -0,0 +1,1170 @@
+            
+            # Initialize benchmark runner
+            benchmark_config = BenchmarkConfig(
+                project_root=self.config.project_root,
+                output_dir=self.config.output_dir,
+                max_workers=self.config.max_workers
+            )
+            self.benchmark_runner = EnhancedBenchmarkRunner(benchmark_config)
+            self.logger.info("Benchmark runner initialized")
+            
+        except Exception as e:
+            self.logger.error(f"Failed to initialize components: {e}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)